tests/app/dao/test_permissions_dao.py

```diff
@@ -5,21 +5,18 @@
 def test_get_permissions_by_user_id_returns_all_permissions(sample_service):
     permissions = permission_dao.get_permissions_by_user_id(user_id=sample_service.users[0].id)
     assert len(permissions) == 8
-    assert (
-        sorted(
-            [
-                "manage_users",
-                "manage_templates",
-                "manage_settings",
-                "send_texts",
-                "send_emails",
-                "send_letters",
-                "manage_api_keys",
-                "view_activity",
-            ]
-        )
-        == sorted([i.permission for i in permissions])
-    )
+    assert sorted(
+        [
+            "manage_users",
+            "manage_templates",
+            "manage_settings",
+            "send_texts",
+            "send_emails",
+            "send_letters",
+            "manage_api_keys",
+            "view_activity",
+        ]
+    ) == sorted([i.permission for i in permissions])
```

Comment:
also just formatting I think?

Reply:
Yes this was a formatting change due to updating black.